.github/renovate.json5

```diff
@@ -1,235 +1,159 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base",
-    ":gitSignOff",
-    "helpers:pinGitHubActionDigests"
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:recommended',
+    ':gitSignOff',
+    'helpers:pinGitHubActionDigests',
   ],
+
   // This ensures that the gitAuthor and gitSignOff fields match
-  "gitAuthor": "renovate[bot] <bot@renovateapp.com>",
-  "includePaths": [
-    ".github/workflows/**",
-    "action.yaml",
-    "defaults/defaults.go",
-    "go.mod",
-    "go.sum",
-    "Dockerfile",
-    "Makefile"
+  gitAuthor: 'renovate[bot] <bot@renovateapp.com>',
+  includePaths: [
+    '.github/workflows/**',
+    'action.yaml',
+    'defaults/defaults.go',
+    'go.mod',
+    'go.sum',
+    'Dockerfile',
+    'Makefile',
   ],
   postUpdateOptions: [
-    "gomodTidy"
+    'gomodTidy',
   ],
-  "pinDigests": true,
-  "ignorePresets": [":prHourlyLimit2"],
-  "separateMajorMinor": true,
-  "separateMultipleMajor": true,
-  "separateMinorPatch": true,
-  "pruneStaleBranches": true,
-  "baseBranches": [
-    "main"
+  pinDigests: true,
+  ignorePresets: [
+    ':prHourlyLimit2',
   ],
-  "vulnerabilityAlerts": {
-    "enabled": true
+  separateMajorMinor: true,
+  separateMultipleMajor: true,
+  separateMinorPatch: true,
+  pruneStaleBranches: true,
+  baseBranches: [
+    'main',
+  ],
+  vulnerabilityAlerts: {
+    enabled: true,
   },
-  "labels": [
-    "renovate/stop-updating",
-    "kind/enhancement",
-    "priority/release-blocker"
+  labels: [
+    'renovate/stop-updating',
+    'kind/enhancement',
+    'priority/release-blocker',
   ],
-  "stopUpdatingLabel": "renovate/stop-updating",
-  "packageRules": [
-    {
-      "groupName": "all github action dependencies",
-      "groupSlug": "all-github-action",
-      "matchPaths": [
-        ".github/workflows/**",
-        "action.yaml"
-      ],
-      "matchUpdateTypes": [
-        "major",
-        "minor",
-        "digest",
-        "patch",
-        "pin",
-        "pinDigest"
-      ]
-    },
+  stopUpdatingLabel: 'renovate/stop-updating',
+  packageRules: [
     {
-      "groupName": "all go dependencies main",
```

Comment:
If possible, we want to keep updating the `github.com/cilium/cilium` dependency in `go.mod` automatically so we probably need to keep this hunk around.

Reply:
good point, readded.